tools/deployment/k8s/s3/deploy.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# This script is tailored for the official AWS CLI container image provided by Amazon (amazon/aws-cli).
+# Prior to running the script, ensure that your AWS credentials or secrets are configured using environment variables.
+# You can set them up in the following ways:
+# 1. Use AWS_CONFIG_FILE to specify the path to a configuration file.
+# 2. Set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY directly as environment variables.
+# 3. Refer to the AWS CLI documentation for additional authentication methods available.
```

Comment:
Can we list out all other required environment variables for clarity?

e.g., "AWS_ENDPOINT_URL" "LOG_VIEWER_BUCKET" "RELEASE_URL"

Reply:
forgot to paste them in